src/Apps/Order/Routes/Payment/index.tsx

```diff
@@ -101,21 +97,6 @@ export const PaymentRoute: FC<PaymentRouteProps> = props => {
 
   const artworkVersion = extractNodes(order.lineItems)[0]?.artworkVersion
 
-  useEffect(() => {
-    if (!bankAccountSelection && selectedPaymentMethod) {
```

Comment:
We are not guarding against non-bank account payment methods, so this hook is updating state for payment methods that don't need it.